doc/plugin_server_ca_memory.md

```diff
@@ -1,13 +1,15 @@
 # Server plugin: ServerCA "memory"
 
-The `memory` plugin implements an in-memory signing authority. No keys are persisted to disk, and
-if the server is restarted, a new signing authority is generated against the upstream CA.
+The `memory` plugin implements an in-memory signing authority. The signing
+keypair is optionally persisted to disk. When the server is loaded, if no
+keypair has been persisted, or if the keypair is expiring/expired, a new
```

Comment:
how do we determine expiring?

Reply:
Same way it is done today. Certificate is "expiring" when it is within 1/6th of the ttl left.